cohort-engine/src/main/java/com/ibm/cohort/engine/cos/CosConfiguration.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * (C) Copyright IBM Corp. 2021, 2021
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.cohort.engine.cos;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.beust.jcommander.DynamicParameter;
+import com.beust.jcommander.Parameter;
+
+public class CosConfiguration {
+
+	@Parameter(names = { "-k", "--cos-api-key" }, description = "COS API key" )
+	private String apiKey;
+
+	@Parameter(names = { "-s", "--cos-id" }, description = "COS service instance id" )
+	private String serviceInstanceId;
+
+	@Parameter(names = { "-u", "--cos-url" }, description = "COS endpoint url" )
+	private String endpointUrl = "https://s3.us-south.cloud-object-storage.appdomain.cloud";
+
+	@Parameter(names = { "-l", "--cos-location" }, description = "COS Location" )
```

Comment:
Will these parameters eventually be hooked in to the MeasureCLI? Wondering if we need to worry about collisions with parameters in MeasureCLI/BaseCLI (ex: https://github.com/Alvearie/quality-measure-and-cohort-service/blob/51762d96ee5e3bc3393701d2a2ddc5afdf3abbb1/cohort-cli/src/main/java/com/ibm/cohort/cli/CohortCLI.java#L58).

Reply:
Correct. 
`JCommander` does throw an `Exception` in that case, but it's at `runtime`.
I can look into adding build-time validation, but I believe the change would not be trivial 
(probably a 50 to 100 line change).

I'll leave this comment unresolved, to see if anyone else wanted to chime in.

Reply:
@rickjstevens, those are all great questions that are outside the scope of this ticket.
@dkwasny-ibm are talking through some of those points that you brought up now,
but I'll make sure we keep your comments in mind during design.